charts/backingservices/README.md

```diff
@@ -39,25 +39,25 @@ Note: Pega does **not** actively update the elasticsearch dependency in `require
-| `domain`                                 | DNS entry associated with externalESDeployment.                                                                                                                                                                                                                                                                                     |
+| Configuration                            | Usage                                                                                                                                                                                                                                                                                                                             |
+|------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `tls`                                    | Setting this to true will make the SRS service use es certificates for connection.                                                                                                                                                                                                                                                |
+| `externalESDeployment.username`          | Give the value of the username as shown in the connection info section of externalESDeployment.                                                                                                                                                                                                                                   |
+| `externalESDeployment.username`          | Give the value of the password as shown in the connection info section of externalESDeployment.                                                                                                                                                                                                                                   |
+| `srsStorage.provisionInternalESCluster`  | <ul><li>Set to `false` to disable this parameter to use your own Elasticsearch service from the SRS cluster; this Requires you to run `$ make external-es-secrets NAMESPACE=<NAMESPACE_USED_FOR_DEPLOYMENT> PATH_TO_CERTIFICATE=<PATH_TO_CERTS>`. </li><li> PATH_TO_CERTIFICATE takes the location value of the placed </li></ul> |
+| `domain`                                 | DNS entry associated with externalESDeployment.                                                                                                                                                                                                                                                                                   |
```

Comment:
Set to provide the DNS entry associated with your external Elasticsearch service set using externalESDeployment.